ansible/testbed-cli.sh

```diff
@@ -228,7 +228,8 @@ function add_topo
         -e ptf_imagename="$ptf_imagename" -e vm_type="$vm_type" -e ptf_ipv6="$ptf_ipv6" \
         $ansible_options $@
 
-  ansible-playbook fanout_connect.yml -i $vmfile --limit "$server" --vault-password-file="${passwd}" -e "dut=$duts" $@
+  ansible-playbook fanout_connect.yml -i $vmfile --limit "$server" --vault-password-file="${passwd}" \
+        -e "dut=$duts" -e ptf_imagename="$ptf_imagename" $@
```

Comment:
instead of passing the ptf_imagename as an argument, add a check here to skip this playbook execution if ptf_imagename has 'apiserver'

Reply:
Implemented the change.